target/linux/ramips/image/rt305x.mk

```diff
@@ -177,8 +177,8 @@ TARGET_DEVICES += dir-620-d1
 define Device/dwr-512-b
   DTS := DWR-512-B
   DEVICE_TITLE := D-Link DWR-512 B
-  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-i2c-core kmod-i2c-ralink kmod-spi-dev \
-			kmod-usb-serial kmod-usb-serial-option kmod-usb-serial-wwan comgt
+  DEVICE_PACKAGES := kmod-spi-dev kmod-usb2 kmod-usb-serial kmod-usb-serial-option \
```

Comment:
Please don't reorder modules... it makes it harder to figure what the substantive change was.